code/modules/mob/living/carbon/human/human.dm

```diff
@@ -40,6 +40,13 @@
 	GLOB.human_list -= src
 	return ..()
 
+/mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+	if(HAS_TRAIT(src, TRAIT_FREERUNNING) && levels <= 2) // falling off one level
```

Comment:
I can't tell if this is a bug but that's how it works right now apparently

Reply:
Seems like a bug, you can test this if you want by putting a breakpoint on Zfall and checking at which point levels goes from 1 to 2 and looking at what is happening at that time. :)

Reply:
Also instead of doing it like this can you make this check the opposite (so check if this is not the case) and then return ..(), leaving the rest of the code unindented